network-services/README.md

```diff
@@ -222,15 +233,26 @@ The goal of the first part of the demo is to show the current state without netw
             +--rw interface?   string
     ```
 
-1. Now let's use the model to configure the fabric.  
-1. Jump back into NSO and enter config mode. 
+1. Now we need to load the VXLAN-EVPN package into NSO. But before loading a package the YANG data module has to be compiled into _fxs_ files that can be loaded into NSO.
+
+    ```
+    make -C packages/vxlan-evpn/src
```

Comment:
What was the reason you needed to do this? I didn't in my example and would prefer that we just include the compiled FXS files in the repo rather than make people compile themselves.

Reply:
The included VXLAN FXS files from the repo were not working for me, hence my need to recompile them.